src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx

```diff
@@ -3,6 +3,7 @@ title: Troubleshooting
 redirect_from:
   - /platforms/javascript/sourcemaps/troubleshooting/
 description: "Troubleshooting for source maps."
+sidebar_order: 8
```

Comment:
Suggest changing the title of this page (but **not** the file name) to "Troubleshooting Source Maps". The reason being that the main Troubleshooting page for JS comes right after this page in the left side navigation and it might be confusing for users to see two identical titles like that.

Reply:
Makes sense to me, updating. 👍